pkg/strategy/rebalance/strategy.go

```diff
@@ -24,9 +25,9 @@ type Strategy struct {
 	QuoteCurrency string           `json:"quoteCurrency"`
 	TargetWeights types.ValueMap   `json:"targetWeights"`
 	Threshold     fixedpoint.Value `json:"threshold"`
+	MaxAmount     fixedpoint.Value `json:"maxAmount"` // max amount to buy or sell per order
+	OrderType     string           `json:"orderType"`
```

Comment:
you can use types.OrderType type directly, it could be unmarshalled

Reply:
ok